internal/action/generate.go

```diff
@@ -524,3 +524,15 @@ func filterPrefix(in []string, prefix string) []string {
 
 	return out
 }
+
+func isStrict(ctx context.Context, c *cli.Context) bool {
+	cfg := config.FromContext(ctx)
+
+	if c.Bool("strict") {
+		return true
+	} else if cfg.IsSet("generate.strict") {
```

Comment:
You don't need the `else` here.

I think you can just remove the else (but keep the if part after moving it to the next line) and the code will work just like now but be a bit more idiomatic.